components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/handlers/SessionDataPublisherImpl.java

```diff
@@ -123,6 +123,19 @@
                         .getUserName(), e);
             }
         }
+        SystemApplicationDAO systemApplicationDAO = new SystemApplicationDAO();
+        try {
+            systemApplicationDTOS = systemApplicationDAO.getApplications(tenantDomain);
+            if (systemApplicationDTOS.length < 0) {
+                if (log.isDebugEnabled()) {
+                    log.debug(
```

Comment:
Can't we remove this new line and keep as a single line?

Reply:
Also add a space before "doesn't"

`"The tenant: " + tenantDomain + " doesn't have any system apps"`

Reply:
resolved